collect_app/src/main/java/org/odk/collect/android/formentry/saving/SaveAnswerFileErrorDialogFragment.java

```diff
@@ -0,0 +1,50 @@
+package org.odk.collect.android.formentry.saving;
+
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.DialogFragment;
+import androidx.lifecycle.ViewModelProvider;
+
+import com.google.android.material.dialog.MaterialAlertDialogBuilder;
+
+import org.odk.collect.android.R;
+import org.odk.collect.android.injection.DaggerUtils;
+
+import javax.inject.Inject;
+
+public class SaveAnswerFileErrorDialogFragment extends DialogFragment {
```

Comment:
This isn't tested at the moment. I've added a note to the "Current Release" project to come back and look at how `DialogFragment` classes are tested and try and backfill this. It seems like the options we have from Android's test tooling is a horrid mess of Robolectric and the new scenario stuff.